docs/docs/account/user-permissions.mdx

```diff
@@ -60,6 +60,27 @@ If your organization has enabled the setting to allow verified users to automati
 
 :::
 
+#### How does the No Access role work?
+
+The `No Access` role is available to organizations enrolled in an Enterprise plan.
+
+In some cases, an organization might want to hide certain projects from a user entirely. This is possible with the `No Access` role. The `No Access` role can either be applied as the user's global role, or it can be a project-specific role.
+
+In the event you want a user to have access to a subset of projects, you can give them a global role of `No Access` and then give project-specific permissions for the projects you want them to be able to view.
+
+If, however, you only want to hide a certain project from a user, you can assign their project-level role of `No Access` which will make it so the user isn't able to view the project.
+
+If you need to apply these rules to many users as once, you can create a Team with the permissions needed, and then add users to that team. But, keep in mind, the user's permissions will be a combination of their user permissions, and the permissions of any team their on, so after adding the user to a team, you may want to reduce their user-level permissions to ensure their previous user-level permissions don't override the permissions inherited by the team(s) they're on.
+
+:::note
+
+Within GrowthBook, not all resources are project-specific. For example, `Attributes`, `Saved Groups`, `Namespaces`, `Environments`, `Presentations`, and `Ideas` all live at the organization level. So, if a user has a global role of `No Access`, they will not be able to view any of these resources. However, if the user has a project-specific role that does
```

Comment:
>  So, if a user has a global role of `No Access`, they will not be able to view any of these resources.

Is this true?  I thought since these resources are not project-scoped, that means everyone can read them.  Did we add read filters to all of these data models too?

Reply:
You are correct - this was from the original docs draft before I wrote any code. This has been updated.